test/over_react/experimental/redux_component_test.dart

```diff
@@ -95,6 +96,31 @@ void main() {
       expect(component.numberOfRedraws, 1);
     });
 
+    test('properly redraws when isPure is true', () async {
+      var store = new Store<BaseState, BaseStateBuilder, BaseActions>(
+        baseReducerBuilder.build(),
+        baseState,
+        baseActions,
+      );
+      var jacket = mount<TestDefaultComponent>((TestDefault()..store = store)());
+      TestDefaultComponent component = jacket.getDartInstance();
+
+      store.actions.trigger1();
+      await new Future.delayed(Duration.ZERO);
+      expect(component.numberOfRedraws, 1);
+
+      jacket.rerender((TestDefault()
+        ..store = store
+        ..id = 'new id'
+      )());
+
+      expect(component.numberOfRedraws, 1);
+
+      component.redraw();
+
+      expect(component.numberOfRedraws, 2);
+    });
```

Comment:
There should also be a test that it updates as expected when `props.store` changes.